tests/unittest/test_loss.py

```diff
@@ -32,3 +33,33 @@ def testTemporalActivationRegularizationLoss():
     print(tar)
     tar(*[mx.nd.arange(1000).reshape(10, 10, 10),
           mx.nd.arange(1000).reshape(10, 10, 10)])
+    
+def testMaskedSoftmaxCrossEntropyLoss():
+    loss_fn = nlp.loss.SoftmaxCEMaskedLoss()
+    pred = mx.nd.array([[[0,0,10],[10,0,0]]]) #N,T,C 1,2,3
+    label = mx.nd.array([[2,2]])
+    valid_length = mx.nd.array([1,]) 
+    loss = loss_fn(pred, label, valid_length)
+    assert loss < 0.1, "1st timestep prediction is correct"
```

Comment:
Should the `assert` message be "...prediction is wrong" here?

Reply:
The first prediction is correct hence the small loss

Reply:
Yes, but the message is printed when the assertion fails

Reply:
Ah I see both of our confusions. Maybe write "1st timestep prediction is correct, but loss was high"?